ChangeLog

```diff
@@ -5,6 +5,20 @@
     modifications to Hashtable that might exposure potential memory corruption 
     with default copy constructor
 
+2015-01-01  Luiz Irber <irberlui@msu.edu>
```

Comment:
Top of the ChangeLog; two spaces after the author name and before the bracketed email address